idea/src/org/jetbrains/kotlin/idea/actions/JavaToKotlinAction.kt

```diff
@@ -131,7 +131,7 @@ class JavaToKotlinAction : AnAction() {
 
             if (enableExternalCodeProcessing && converterResult!!.externalCodeProcessing != null) {
                 val question = "Some code in the rest of your project may require corrections after performing this conversion. Do you want to find such code and correct it too?"
-                if (!askExternalCodeProcessing || (Messages.showOkCancelDialog(project, question, title, Messages.getQuestionIcon()) == Messages.OK)) {
+                if (!askExternalCodeProcessing || (Messages.showYesNoDialog(project, question, title, Messages.getQuestionIcon()) == Messages.OK)) {
```

Comment:
Here and below, I suppose you should compare the result of `showYesNoDialog` with `Messages.YES`, not with `Messages.OK`. According to `showYesNoDialog` contract, it can return `YES`, `NO` or `CANCEL`, and never returns `OK`.

Reply:
You are right, I missed that. Fixed and pushed. Thanks @mglukhikh